config/sample_manifest_excel/manifest_types.yml

```diff
@@ -678,3 +678,11 @@ heron:
     - :donor_id2
     - :control_type
     - :priority
+cardinal:
+  heading: "Cardinal"
+  asset_type: "1dtube"
+  columns:
+    - :sanger_tube_id
+    - :sanger_sample_id
+    - :supplier_name
+    - :volume
```

Comment:
OK so:
- `:sanger_tube_id` -> this is auto-populated with the NT barcode, and can be pasted over for the UKB tubes, which have their own barcodes.
- `:sanger_sample_id` -> this is auto-populated
- `:supplier_name` -> this will be manually populated with the NT barcode or UKB barcode
- `:volume` -> this will be manually filled out - actually not sure if this is something that is supplied or will be measured by the bioresource team - will check

And then supplier will be filled in when generating the manifest, as per your comment on the story.

Is that correct?

Reply:
Volume - Stef reckons this is measured on-site by the bioresource team - is this field the one you said is 'customer measured volume'? If so, might be misleading.

Reply:
Yep, saw the slack conversation, it might be that this gets dropped. But essentially that means we have no customer supplied fields, so it might be easier it the manifest registers all samples up-front. Plus sets the supplier name. That way there would be no need to upload manifests at any stage. A bit more work for us, but far less work for SSRs or anyone else. It does depend on how the volume conversation resolves though.